lib/hammer_cli_foreman/compute_resources/ec2.rb

```diff
@@ -0,0 +1,19 @@
+module HammerCLIForeman
+  module ComputeResources
+    class EC2 < Base
```

Comment:
please don't change the path of this files, 
there are plugins that depend on these paths, and this is not so good to change those (for example hammer_cli_foreman_kubevirt)